code/game/objects/items/weapons/design_disks/NeoTheology.dm

```diff
@@ -251,7 +246,7 @@
 		/datum/design/autolathe/cell/medium/high,
 	)
 
-/obj/item/computer_hardware/hard_drive/portable/design/guns/nt_valkirye
+/obj/item/computer_hardware/hard_drive/portable/design/guns/nt_valkyrie
```

Comment:
If this path is used anywhere else it will need to be updated.

Reply:
It's not.